subscription_oca/__manifest__.py

```diff
@@ -10,6 +10,7 @@
     "license": "AGPL-3",
     "author": "Domatix, Odoo Community Association (OCA)",
     "depends": ["sale_management", "account"],
+    "maintainers": ["ilyasProgrammer"],
```

Comment:
changing maintainer can be a separated commit that you can backport/fwd port easily ;)